tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -146,22 +175,6 @@ describe('Header', () => {
     expect(profileTab.hasClass('usa-current')).toEqual(true)
   })
 
-  it("should add usa-current class to Admin tab when on '/admin'", () => {
```

Comment:
Is this test no longer relevant?

Reply:
Hmm it's relevant but seems to fail now.

Reply:
I'm thinking it's failing because Jest doesn't know about `process.env.REACT_APP_BACKEND`

Reply:
Which should be `REACT_APP_BACKEND_URL` now

Reply:
Yeah not sure why that's in the path at all? It should just be `/admin` yeah? https://github.com/raft-tech/TANF-app/blob/e165e08f9b933d07852e49560fcf0c91bcc70048/tdrs-frontend/src/components/Header/Header.jsx#L64

Reply:
Changing the `href` to just `/admin` makes the test pass when I re-add it.

Reply:
Ah yeah, the backend URL definitely shouldn't be used there. Good catch!

Reply:
Just wanted to add a correction to my last comment - in this case we should be using the backend URL and this test is not relevant anymore.